skyvern/webeye/utils/page.py

```
@@ -540,3 +540,175 @@ async def wait_for_animation_end(self, timeout_ms: float = 3000) -> None:
+        Checks are ordered by timeout (highest first) so the longest timeout
+        acts as the primary upper bound when checks complete early.
+
+        This is designed for cached action execution to ensure the page is ready
+        before attempting to interact with elements.
+        """
+        # 1. Wait for loading indicators to disappear (longest timeout first)
+        try:
+            await self._wait_for_loading_indicators_gone(timeout_ms=loading_indicator_timeout_ms)
+        except (TimeoutError, asyncio.TimeoutError):
+            LOG.warning("Loading indicator timeout - some indicators may still be present, proceeding")
+        except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)